index.html

```diff
@@ -0,0 +1,514 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/index.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,500;0,600;1,500&display=swap"
+      rel="stylesheet"
+    />
+    <title>BEM practice | KMarketer</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="nav">
+        <h1
+          class="title title--letter-spacing nav__title nav__title--left-spacing"
+        >
+          KMarketer
+        </h1>
+        <ul class="menu menu--flex">
+          <li class="menu__item">Home</li>
+          <li class="menu__item">Products</li>
+          <li class="menu__item">Pricing</li>
+          <li class="menu__item">Contact</li>
+          <li class="menu__item menu__item--flex">
+            <img src="./images/seach-icon.svg" />
+            <img src="./images/shopping-card-icon.svg" />
+          </li>
+          <li>
+            <img src="./images/menu-icon.svg" />
+          </li>
+        </ul>
+      </nav>
+
+      <!-- nav on mobile screen -->
+      <nav class="mobile-nav">
+        <div class="mobile-nav__header">
+          <h1
+            class="title title--letter-spacing mobile-nav__title--left-spacing"
```

Comment:
`mobile-nav__title--left-spacing` is included without it's default Element `mobile-nav__title`
I expect to see something like this `class="title title--letter-spacing mobile-nav__title mobile-nav__title--left-spacing"`